### ISSUE TITLE
False positive on no-array-for-each

### ISSUE THREAD

**Issue by sfabriece** [sindresorhus/eslint-plugin-unicorn#1087]
React.Children.forEach is being wrongly flagged.

`no-array-for-each`

```js
React.Children.forEach(children, (child) => {<child />});
```
